apps/rush/src/utilities/PackageReviewChecker.ts

```diff
@@ -30,7 +27,8 @@ export default class PackageReviewChecker {
   }
 
   public saveFile(): void {
-    this._packageReviewConfiguration.saveFile(this._rushConfiguration.packageReviewFile);
+    this._rushConfiguration.browserApprovedPackages.saveToFile();
```

Comment:
>browserApprovedPackages [](start = 28, length = 23)

should this just be under an object called "approvedPackages" ?? it would make `approvedPackagesIgnoredNpmScopes` more understandable